src/app/api/api_v1/endpoints/detections.py

```diff
@@ -134,6 +139,18 @@ async def create_detection(
                 org = cast(Organization, await organizations.get(token_payload.organization_id, strict=True))
                 if org.telegram_id:
                     background_tasks.add_task(telegram_client.notify, org.telegram_id, det.model_dump_json())
+
+            if slack_client.is_enabled:
+                org = cast(Organization, await organizations.get(token_payload.organization_id, strict=True))
```

Comment:
Let's avoid the double call if telegram and slack client are enabled

Reply:
hum I don't understand, what is the issue here ?

Reply:
ok sorry I wrongly understand the first time